__tests__/app.test.js

```diff
@@ -152,27 +162,27 @@ describe("PATCH /api/articles/:article_id", () => {
         expect(body.msg).toBe("No article found for article_id: 199");
       });
   });
+});
 
-  describe("GET /api/users", () => {
-    test("should return status 200", () => {
-      return request(app).get("/api/users").expect(200);
-    });
-
-    test("should return an array of objects", () => {
-      return request(app)
-        .get("/api/users")
-        .then(({ body }) => {
-          const { users } = body;
-          expect(users).toBeInstanceOf(Array);
-          expect(users).toHaveLength(4);
-          users.forEach((user) => {
-            expect(user).toEqual(
-              expect.objectContaining({
-                username: expect.any(String),
-              })
-            );
-          });
+describe("GET /api/users", () => {
+  test("should return status 200", () => {
```

Comment:
Combine this test with the one below - you can check for the correct status and the correct response in one test. No need to separate them out.

Reply:
Done